app/styles/components/map/c-layer-panel.scss

```diff
@@ -0,0 +1,68 @@
+@import '../../settings';
+
+.c-layer-panel {
+  padding: 5px 0 0;
+
+  .layer-list {
+
+    > .layer-item {
+      cursor: pointer;
+
+      display: flex;
+      justify-content: space-between;
+
+      &:not(:first-child) {
+        margin: 12px 0 0;
+      }
+
+      .switcher {
+        -webkit-appearance: none;
+        -webkit-transition: .15s;
+        background: linear-gradient(to bottom, $hsla-begin, $hsla-end), linear-gradient(to right, $color-27 50%, currentColor 50%) 0 0, 5px 0;
+        background-size: 100% 100%, 200% 100%;
+        border-radius: 25px;
+        cursor: pointer;
+        height: 10px;
+        margin: 0 8px 0 0;
+        outline: 0;
+        padding: 0 10px 0 0;
+        position: relative;
+        width: 20px;
+
+        &::after {
+          background: $color-26 -webkit-linear-gradient($hsla-end, $hsla-begin);
```

Comment:
We don't need any prefix here. Can you use the same syntax as you did on line 21?